### DISCUSSION TITLE
GH actions all jobs are in queued from long time

### DISCUSSION THREAD

**Discussion by AdityaMehtaIselect** [community/community#143054] [Actions] [answered]
### Select Topic Area

Question

### Body

All my GH action jobs are Queued from long time

**Comment by jfunction** [comment]
https://www.githubstatus.com/

**Comment by samus-aran** [comment, marked as answer]
Hi @AdityaMehtaIselect  👋 

Thanks for sharing and we appreciate your patience. We confirm there was an incident with Actions which has now been resolved. [Latest update is available here.](https://github.com/orgs/community/discussions/143045#discussioncomment-11098688)
